test/types/optionSelect.test.js

```diff
@@ -10,27 +10,54 @@ const field = 'is_open'
   , expectedLabel = 'Is Open'
   , type = 'optionSelect'
   , fieldSets = [[{ field, type, optionType }]]
+  , model = { is_open: ['true'] }
   ;
 
 describe('optionSelect', () => {
   it('Renders', async () => {
     const props = {
         cardConfig: { fieldSets },
-        model: { is_open: 'true' },
+        model,
       };
 
     const tester = await new Tester(Card, { props }).mount();
     expect(tester.text()).toContain(expectedLabel);
+    expect(tester.text()).toContain('Yes');
   });
 
   it('Edits', async () => {
     const onSave = jest.fn()
       , props = {
         cardConfig: { fieldSets },
+        model,
         onSave,
       };
 
     const tester = await new Tester(FormCard, { props }).mount();
     expect(tester.text()).toContain(expectedLabel);
   });
+
+  it('Handles array with single value (useful for summary tables)', async () => {
+    const props = {
+        cardConfig: { fieldSets },
+        model: { is_open: ['true'] },
```

Comment:
is this still necessary? since it's defined in the outer const
```suggestion
        model: { is_open: ['true'] },
```

Reply:
Good catch. the outterone should have been no-array.